project-koku/koku
# COST-2993 - OCI cache not invalidated

## Jira Ticket

[COST-2993](https://issues.redhat.com/browse/COST-2993)

## Description

This change will invalidate cache for OCI

## Testing

1. Checkout Branch
2. Restart Koku
3. launch shell and run `make create-test-customer` and `make load-test-customer-data test_source=oci`
4. Hit endpoint  http://127.0.0.1:8000/api/cost-management/v1/reports/oci/costs/ during oci data ingestion
    1. You should see no data, this is expected.
6. Wait for the data ingestion to complete and hit the endpoint again, you should see data in the report.

## Notes

...
